model/context.go

```diff
@@ -107,10 +107,37 @@ func truncate(s string) string {
 	return s
 }
 
+type NetworkInfo struct {
+	EffectiveType *string
+	RoundTripTime *int64
+	Downlink      *int64
+	DownlinkMax   *int64
```

Comment:
Downlink could be fractional, so should be a float64:

> The downlink attribute represents the effective bandwidth estimate in megabits per second, rounded to nearest multiple of 25 kilobits per second

Same for downlinkMax.

Reply:
my understand was that rum would not send decimal number anyways. will change..